### ISSUE TITLE
simplified-installer fails installing yq RHEL-8.6.0-updates-20230404.4

### ISSUE THREAD

**Issue by mcattamoredhat** [virt-s1/rhel-edge#2571]
Command `sudo pip3 install -v yq fails` Collecting tomlkit>=0.11.7 (from yq)

tomlkit-0.11.7 requires-python:>=3.7

```
+ sudo pip3 install yq
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Collecting yq
  Using cached https://files.pythonhosted.org/packages/a1/02/df27a89765c3ff839324a5cb8cc7c2e020c917dd030ba9d29bd2d9998ae3/yq-3.2.0-py3-none-any.whl
Requirement already satisfied: argcomplete>=1.8.1 in /usr/lib/python3.6/site-packages (from yq)
Collecting tomlkit>=0.11.7 (from yq)
  Could not find a version that satisfies the requirement tomlkit>=0.11.7 (from yq) (from versions: 0.1.0, 0.1.1, 0.2.0, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.4.5, 0.4.6, 0.5.0, 0.5.1, 0.5.2, 0.5.3, 0.5.4, 0.5.5, 0.5.6, 0.5.7, 0.5.8, 0.5.9, 0.5.10, 0.5.11, 0.6.0, 0.7.0, 0.7.1, 0.7.2, 0.8.0, 0.9.0, 0.9.1, 0.9.2, 0.10.0, 0.10.1, 0.10.2, 0.11.0, 0.11.1, 0.11.2, 0.11.3, 0.11.4, 0.11.5, 0.11.6)
No matching distribution found for tomlkit>=0.11.7 (from yq)
```

Verbosed output:

```
$ sudo pip3 install -v yq
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Collecting yq
  1 location(s) to search for versions of yq:
  * https://pypi.python.org/simple/yq/
  Getting page https://pypi.python.org/simple/yq/
  Looking up "https://pypi.python.org/simple/yq/" in the cache
  Returning cached "301 Moved Permanently" response (ignoring date and etag information)
  Looking up "https://pypi.org/simple/yq/" in the cache
  Current age based on date: 148
  Freshness lifetime from max-age: 600
  Freshness lifetime from request max-age: 600
  The response is "fresh", returning cached response
  600 > 148
  Analyzing links from page https://pypi.org/simple/yq/
    Skipping link https://files.pythonhosted.org/packages/2e/a5/3555543c43e70feb018e900838c29487a25b9fb68ed9ee4eeb83388aef9b/yq-0.0.2-py2-none-any.whl#sha256=7eeb59d2822d611962dee9d94a2e0515b58ff125c2c6ad268716ad64b99f40e8 (from https://pypi.org/simple/yq/); it is not compatible with this Python
    Found link https://files.pythonhosted.org/packages/16/13/59ac0722672eee7e3da3c10c5ef3ad94ae712a34851196a8e97a52d8e3e5/yq-0.0.2.tar.gz#sha256=f912755290e3155312b1963f9b03453416344e0a32e4a294c00e928d58460766 (from https://pypi.org/simple/yq/), version: 0.0.2
[...]   
    Found link https://files.pythonhosted.org/packages/2b/df/971fa5db3250bb022105d17f340339370f73d502e65e687a94ca1a4c4b1f/tomlkit-0.11.6-py3-none-any.whl#sha256=07de26b0d8cfc18f871aec595fda24d95b08fef89d147caa861939f37230bf4b (from https://pypi.org/simple/tomlkit/) (requires-python:>=3.6), version: 0.11.6
    Found link https://files.pythonhosted.org/packages/ff/04/58b4c11430ed4b7b8f1723a5e4f20929d59361e9b17f0872d69681fd8ffd/tomlkit-0.11.6.tar.gz#sha256=71b952e5721688937fb02cf9d354dbcf0785066149d2855e44531ebdd2b65d73 (from https://pypi.org/simple/tomlkit/) (requires-python:>=3.6), version: 0.11.6
    The package https://files.pythonhosted.org/packages/c0/83/eb757ef200543637c40f136e370ef05158d4079ad61da2cf455fe34c508d/tomlkit-0.11.7-py3-none-any.whl#sha256=5325463a7da2ef0c6bbfefb62a3dc883aebe679984709aee32a317907d0a8d3c (from https://pypi.org/simple/tomlkit/) (requires-python:>=3.7) is incompatible with the pythonversion in use. Acceptable python versions are:>=3.7
    The package https://files.pythonhosted.org/packages/4d/4e/6cb8a301134315e37929763f7a45c3598dfb21e8d9b94e6846c87531886c/tomlkit-0.11.7.tar.gz#sha256=f392ef70ad87a672f02519f99967d28a4d3047133e2d1df936511465fbb3791d (from https://pypi.org/simple/tomlkit/) (requires-python:>=3.7) is incompatible with the pythonversion in use. Acceptable python versions are:>=3.7
  Could not find a version that satisfies the requirement tomlkit>=0.11.7 (from yq) (from versions: 0.1.0, 0.1.1, 0.2.0, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.4.4, 0.4.5, 0.4.6, 0.5.0, 0.5.1, 0.5.2, 0.5.3, 0.5.4, 0.5.5, 0.5.6, 0.5.7, 0.5.8, 0.5.9, 0.5.10, 0.5.11, 0.6.0, 0.7.0, 0.7.1, 0.7.2, 0.8.0, 0.9.0, 0.9.1, 0.9.2, 0.10.0, 0.10.1, 0.10.2, 0.11.0, 0.11.1, 0.11.2, 0.11.3, 0.11.4, 0.11.5, 0.11.6)
Cleaning up...
No matching distribution found for tomlkit>=0.11.7 (from yq)
Exception information:
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/lib/python3.6/site-packages/pip/commands/install.py", line 346, in run
    requirement_set.prepare_files(finder)
  File "/usr/lib/python3.6/site-packages/pip/req/req_set.py", line 381, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/usr/lib/python3.6/site-packages/pip/req/req_set.py", line 557, in _prepare_file
    require_hashes
  File "/usr/lib/python3.6/site-packages/pip/req/req_install.py", line 278, in populate_link
    self.link = finder.find_requirement(self, upgrade)
  File "/usr/lib/python3.6/site-packages/pip/index.py", line 535, in find_requirement
    'No matching distribution found for %s' % req
pip.exceptions.DistributionNotFound: No matching distribution found for tomlkit>=0.11.7 (from yq)

```

This issue also affects RHEL-8.8.0-20230403.3 and possibly more distros.

**Comment by mcattamoredhat** [comment]
This issue also affects edge-simplified-installer CentOS-Stream-8-20230404.0

**Comment by mcattamoredhat** [comment]
Fixed in RHEL-8-6 by compose RHEL-8.6.0-updates-20230405.5. Version of tomlkit has been changed to 0.11.7

```
+ sudo pip3 install yq
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.
Collecting yq
  Downloading https://files.pythonhosted.org/packages/99/3f/bb40b896d62750c6c66ba3a1768656091bed94e083b14d9b320042ac60e0/yq-3.2.1-py3-none-any.whl
Requirement already satisfied: argcomplete>=1.8.1 in /usr/lib/python3.6/site-packages (from yq)
Collecting tomlkit>=0.11.6 (from yq)
  Downloading https://files.pythonhosted.org/packages/2b/df/971fa5db3250bb022105d17f340339370f73d502e65e687a94ca1a4c4b1f/tomlkit-0.11.6-py3-none-any.whl
[...]
```

Results of next CentOS-Stream-8 compose will be monitored.
